client/lib/pages/onboarding/country_list_page.dart

```diff
@@ -27,30 +27,49 @@ class CountryListPage extends StatefulWidget {
 
 class _CountryListPageState extends State<CountryListPage> {
   String selectedCountryCode;
+  var listData;
 
   @override
   void initState() {
     super.initState();
-
     selectedCountryCode = widget.selectedCountryCode;
+    listData = (widget.countries?.values ?? []);
   }
 
   @override
   Widget build(BuildContext context) {
     return PageScaffold(
       // TODO: localize?
-      title: 'Country',
+      // title: 'Country',
+      headerWidget: TextField(
+          decoration: InputDecoration(
+              border: InputBorder.none, hintText: 'Enter your country'),
+          onChanged: (String value) async {
+            List filtered = (widget.countries?.values ?? []).where((element) {
+              var listItem = element.name
```

Comment:
Use `nameMatch` as that better indicates the purpose. Using `listItem` suggests a pointer to an object not a boolean.
```suggestion
              var nameMatch = element.name
```